packages/terra-clinical-site/src/examples/alert/Index.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-clinical-alert/docs/README.md';
+import { version } from 'terra-clinical-alert/package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import AlertSrc from '!raw-loader!terra-clinical-alert/src/Alert';
+
+// Example Files
+import AlertExample from './AlertExample';
+import ErrorExample from './ErrorExample';
+import WarningExample from './WarningExample';
+import GapCheckingExample from './GapCheckingExample';
+import AdvisoryExample from './AdvisoryExample';
+import InfoExample from './InfoExample';
+import SuccessExample from './SuccessExample';
+import OutsideRecordsExample from './OutsideRecordsExample';
+import CustomExample from './CustomExample';
+import LongTextExample from './LongTextExample';
+import ActionExample from './ActionExample';
+import SmallContainerExample from './SmallContainerExample';
+import DismissibleExample from './DismissibleExample';
+import ActionAndDismissibleExample from './ActionAndDismissibleExample';
+import BuildYourOwnAlertExample from './BuildYourOwnAlertExample';
+
+const AlertExamples = () => (
```

Comment:
I included all of the alert examples because the clinical-alert is intended have all of the base alert types plus the additional clinical alert. I wanted consumers to know that these are all available with terra-clinical-alert and that they wouldn't need to consume terra-alert separately to get the other alert types.